MyAutoAPI1/DTO/Identity/GetAlluser/GetAllsuer.cs

```diff
@@ -0,0 +1,22 @@
+﻿namespace MyAutoAPI1.DTO.Identity.GetAlluser
+{
+    public class GetAlluser : IGetAlluser
```

Comment:
UserDTO

Reply:
https://www.infoworld.com/article/3562271/how-to-use-data-transfer-objects-in-aspnet-core-31.html#:~:text=A%20Data%20Transfer%20Object%20(commonly,back%20to%20the%20presentation%20layer.